src/website/static/admin/css/main.css

```diff
@@ -0,0 +1,20 @@
+/* Override Bootstrap primary color if needed */
+.text-primary {
+  color: #0056b3 !important;
+}
+

```

Comment:
Hi @OchiengPaul442 Is CSS meant for a backend repo?

Reply:
the added CSS is just to further customize the default Django admin panel